File: SDKs/Aspose.Email_Cloud_SDK_for_CSharp/src/Com/Aspose/Email/Model/EmailProperty.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Email.Model {
  public class EmailProperty {
    public Link Link { get; set; }

    public string Name { get; set; }

    public object Value { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class EmailProperty {\n");
      sb.Append("  Link: ").Append(Link).Append("\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Value: ").Append(Value).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Email-Cloud-SDK-for-Android/Aspose-Email-Cloud-SDK-for-Android/src/main/java/com/aspose/email/model/EmailProperty.java
package com.aspose.email.model;

import com.aspose.email.model.Link;
public class EmailProperty {
  private Link Link = null;
  private String Name = null;
  private Object Value = null;
  /**
	 * getLink
	 * Gets Link
	 * @return Link
	 */
  public Link getLink() {
    return Link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param Link Link
	 */
  public void setLink(Link Link) {
    this.Link = Link;
  }

  /**
	 * getName
	 * Gets String
	 * @return Name
	 */
  public String getName() {
    return Name;
  }

	/**
	 * setName
	 * Sets String
	 * @param Name String
	 */
  public void setName(String Name) {
    this.Name = Name;
  }

  /**
	 * getValue
	 * Gets Object
	 * @return Value
	 */
  public Object getValue() {
    return Value;
  }

	/**
	 * setValue
	 * Sets Object
	 * @param Value Object
	 */
  public void setValue(Object Value) {
    this.Value = Value;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class EmailProperty {\n");
    sb.append("  Link: ").append(Link).append("\n");
    sb.append("  Name: ").append(Name).append("\n");
    sb.append("  Value: ").append(Value).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

